app/frontend/src/services/formService.js

```diff
@@ -264,16 +264,20 @@ export default {
    * @param {string} minDate The form uuid
    * @param {string} maxDate The form uuid
    * @param {string} format The file format csv or json
+   * @param {string} showDeleted should deleted submissions be included in export? true or false
+   * @param {string} showDrafts should draft submissions be included in export? true or false
    * @returns {Promise} An axios response
    */
-  exportSubmissions(formId, minDate, maxDate, format) {
+  exportSubmissions(formId, minDate, maxDate, format, showDeleted, showDrafts) {
     return appAxios().get(`${ApiRoutes.FORMS}/${formId}/export`,
       {
         params: {
           format: format,
           type: 'submissions',
           minDate: minDate ? minDate : undefined,
-          maxDate: maxDate ? maxDate : undefined
+          maxDate: maxDate ? maxDate : undefined,
+          showDeleted: showDeleted ? showDeleted : false,
+          showDrafts: showDrafts ? showDrafts : false
         },
         responseType: 'blob'
       }
```

Comment:
The exportSubmissions function is starting to become a bit hefty with function arguments. General ES conventions usually recommend to not have more than 3 or 4 arguments to a function call - any more than that and we'll want to start considering passing in options as an object argument instead of explicitly as separated arguments. So we could do something like the following instead:
```js
exportSubmissions(formId, format, options = {}) {
  return appAxios().get(`${ApiRoutes.FORMS}/${formId}/export`,
    {
      params: {
        format: format,
        type: 'submissions',
        ...options
      },
      responseType: 'blob'
    }
  );
},
```
This would give us the liberty of handling optionality of the minDate, maxDate, showDrafts and showDeleted attributes. If they're not defined in the object, destructuring by default will have them be undefined, and not show up in the network query request at all - a much cleaner approach.